src/tls/Makefile.Kremlin

```diff
@@ -4,21 +4,24 @@ build: build-internal-test build-library build-external-test
 
 test: test-internal-test # test-library test-external-test
 
-clean: clean-internal-test clean-library clean-external-test
+ifneq (,$(MITLS_USE_OPENSSL))
+CONCRETE_FLAVOR	= -OpenSSL
```

Comment:
if you want to get rid of $(CONCRETE_FLAVOR), I'm fine with leaving just $(FLAVOR) and in the actual invocation of `fstar`, doing `fstar --codegen $(patsubst -%,,$(FLAVOR))`

Reply:
I'd like to avoid the exponential explosion of `#flavors ** #concrete-flavors`. I though about adding a new `CODEGEN` variable, but I think I like your `patsubst` better!

Reply:
do you want to do it before the merge then?